.github/workflows/container-image-buildah.yml

```diff
@@ -25,11 +42,18 @@ jobs:
   buildah:
     runs-on: ubuntu-latest
     steps:
+      - name: Sanitize Platforms
+        id: platforms
+        run: |
+          platforms="${{ inputs.platforms == '' && 'linux/amd64' || inputs.platforms }}"
+          archs="$( sed -e 's#linux/##g' <<< $platforms )"
+          echo "platforms=$platforms" >> $GITHUB_OUTPUT
+          echo "archs=$archs" >> $GITHUB_OUTPUT
```

Comment:
Do we need all of this just because clang is broken in s390x. Couldn't we just use gcc?

Reply:
This part just selects platforms to build for. It's not because of s390 and will be equally needed if gcc was used. Presently build is too slow to complete for non-native platforms regardless of compiler. So we can only build for x86 in github. If we find no solution, we may need to use circleci in case we want community image to support all archs.

Reply:
btw the s390x and ppc64le clang builda were fixed upstream